specification/playwrighttesting/PlaywrightTesting.AuthManager/main.tsp

```diff
@@ -189,23 +223,54 @@ alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
 
 interface AccessTokens {
   // AccessToken Operations
+  @doc("Creates an access-token for the account with given access-token id. Authorization required is Bearer JWT Access token provided by EntraID.")
+  createOrReplace is Operations.ResourceCreateOrReplace<AccessToken>;
 
-  #suppress "@azure-tools/typespec-azure-core/use-standard-names" "Existing name"
-  @doc("Creates an access-token with given access-token id in guid format.")
-  createAccessToken is Operations.ResourceCreateOrReplace<AccessToken>;
+  @doc("Gets an access-token for the account with given access-token id. Authorization required is Bearer JWT Access token provided by EntraID.")
+  get is Operations.ResourceRead<AccessToken>;
 
-  @doc("Get an access-token for given access-token id in guid format.")
-  getAccessToken is Operations.ResourceRead<AccessToken>;
+  @doc("Deletes an access-token for the account with given access-token id. Authorization required is Bearer JWT Access token provided by EntraID.")
+  delete is Operations.ResourceDelete<AccessToken>;
 
-  @doc("Revoke an access-token with given access-token id in guid format.")
-  deleteAccessToken is Operations.ResourceDelete<AccessToken>;
-
-  @doc("List access-tokens for a given accountId.")
-  listAccessTokens is Operations.ResourceList<AccessToken>;
+  @doc("Lists access-tokens for the given account id. It can use OData query params like $select, $filter, $orderby, $top and $skip. The default page size is 10. Use nextLink in response to fetch more objects in the list. Authorization required is Bearer JWT Access token provided by EntraID.")
+  list is Operations.ResourceList<AccessToken>;
 }
 
+alias GetBrowsersParams = {
+  @query
+  @doc("The run id provided by client for corresponding remote test run.")
+  runId?: string;
+
+  @query
+  @doc("The os provided by client for remote test runs.")
+  os?: OS = OS.Linux;
+
+  @path
+  @doc("The account id.")
+  accountId: string;
```

Comment:
I am not sure how much it effects the Swagger or API. But I'd expect the most important parameter (I assume it be `accountId`) at the top.

Reply:
Let me do this.

Reply:
Done